Created Output Log File: /workdir/tasks_run/data/nfb_logs/p004_calculator_script_20240912_16h40m52s.txtexperimenter screen width: 3390
experimenter screen height: 1080
Second monitor resolution: 1920x1080
TO SHOW INSTRUCTIONS, PLEASE PRESS 'r'.
Showing Instructions. Task will start when 's' is pressed (scanner key maps to s on mac, so starting the experiment will trigger the task)
Running Main Calculation Script ... 
Starting Block1 ... 
========================================
Starting Block1, Trial 1... 
========================================
Waiting For New File ...
New File Found In Dir...
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/76644678530720355580.dcm
Mean ROI Activation: 269.34625244140625
length design matrix: 1
  trial_type  onset  duration
0       rest    0.0      1.06
Error:
Cannot set a DataFrame with multiple columns to the single column trial1
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/ScriptManager.py", line 28, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 26, in run_trial
    dictionary = Calculator.get_resid(dictionary=dictionary, block=block, trial=trial)
  File "/workdir/tasks_run/scripts/Calculator.py", line 79, in get_resid
    dictionary[f"block{block}"]["event_dict"][f"trial{trial}"] = events  # update in data dictionary
  File "/usr/local/lib/python3.10/site-packages/pandas/core/frame.py", line 4301, in __setitem__
    self._set_item_frame_value(key, value)
  File "/usr/local/lib/python3.10/site-packages/pandas/core/frame.py", line 4459, in _set_item_frame_value
    raise ValueError(
ValueError: Cannot set a DataFrame with multiple columns to the single column trial1

Retries left: 3
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/76644678530720355580.dcm
Mean ROI Activation: 269.34625244140625
length design matrix: 1
  trial_type  onset  duration
0       rest    0.0      1.06
Error:
Cannot set a DataFrame with multiple columns to the single column trial1
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/ScriptManager.py", line 28, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 26, in run_trial
    dictionary = Calculator.get_resid(dictionary=dictionary, block=block, trial=trial)
  File "/workdir/tasks_run/scripts/Calculator.py", line 79, in get_resid
    dictionary[f"block{block}"]["event_dict"][f"trial{trial}"] = events  # update in data dictionary
  File "/usr/local/lib/python3.10/site-packages/pandas/core/frame.py", line 4301, in __setitem__
    self._set_item_frame_value(key, value)
  File "/usr/local/lib/python3.10/site-packages/pandas/core/frame.py", line 4459, in _set_item_frame_value
    raise ValueError(
ValueError: Cannot set a DataFrame with multiple columns to the single column trial1

Getting New Dicom Instead...
Waiting For New File ...
New File Found In Dir...
Retries left: 2
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/26756768551808854351.dcm
Mean ROI Activation: 269.34625244140625
length design matrix: 1
  trial_type  onset  duration
0       rest    0.0      1.06
Error:
Cannot set a DataFrame with multiple columns to the single column trial1
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/ScriptManager.py", line 28, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 26, in run_trial
    dictionary = Calculator.get_resid(dictionary=dictionary, block=block, trial=trial)
  File "/workdir/tasks_run/scripts/Calculator.py", line 79, in get_resid
    dictionary[f"block{block}"]["event_dict"][f"trial{trial}"] = events  # update in data dictionary
  File "/usr/local/lib/python3.10/site-packages/pandas/core/frame.py", line 4301, in __setitem__
    self._set_item_frame_value(key, value)
  File "/usr/local/lib/python3.10/site-packages/pandas/core/frame.py", line 4459, in _set_item_frame_value
    raise ValueError(
ValueError: Cannot set a DataFrame with multiple columns to the single column trial1

Getting New Dicom Instead...
Waiting For New File ...
New File Found In Dir...
Retries left: 1
Using DICOM:/workdir/tasks_run/data/sambashare/test_dir/10105919489903939851.dcm
Mean ROI Activation: 269.34625244140625
length design matrix: 1
  trial_type  onset  duration
0       rest    0.0      1.06
Error:
Cannot set a DataFrame with multiple columns to the single column trial1
Traceback (most recent call last):
  File "/workdir/tasks_run/scripts/ScriptManager.py", line 28, in wrapper
    updated_dictionary = func(*args, **kwargs)
  File "/workdir/tasks_run/scripts/nf_calc_MW.py", line 26, in run_trial
    dictionary = Calculator.get_resid(dictionary=dictionary, block=block, trial=trial)
  File "/workdir/tasks_run/scripts/Calculator.py", line 79, in get_resid
    dictionary[f"block{block}"]["event_dict"][f"trial{trial}"] = events  # update in data dictionary
  File "/usr/local/lib/python3.10/site-packages/pandas/core/frame.py", line 4301, in __setitem__
    self._set_item_frame_value(key, value)
  File "/usr/local/lib/python3.10/site-packages/pandas/core/frame.py", line 4459, in _set_item_frame_value
    raise ValueError(
ValueError: Cannot set a DataFrame with multiple columns to the single column trial1

Getting New Dicom Instead...
Waiting For New File ...
New File Found In Dir...
Ran out of retries. Skipping this trial.
Ending Block1, Trial 1... 
========================================
Starting Block1, Trial 2... 
========================================
Waiting For New File ...
---- Keyboard Interrupt Detected ----
What Would You Like to Do?
(1) Continue With The Block
(2) End The Session
(3) Start New Block
dicom dir using: /workdir/tasks_run/data/sambashare/test_dir
